tidb-server/main.go

```diff
@@ -259,15 +254,15 @@ func setCPUAffinity() {
 			c, err := strconv.Atoi(af)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "wrong affinity cpu config: %s", *affinityCPU)
-				exit()
+				os.Exit(1)
```

Comment:
This change removes the `syncLog()` call, is that intended?

Reply:
yes. I think no need to `syncLog` here. If it's necessary, many other places need too.

Reply:
+1, sync on stdout/stderr seems be useless https://github.com/uber-go/zap/issues/370#issuecomment-307152090, maybe we can delete them(both in linux and newest macos)